<!-- -->
<launch>
    <arg name="config_path" default="/home/rn/robotconfig/system_param.yaml" />
        <node pkg="path_navi" type="path_navi_server" name="path_navi_server" output="screen">
        <param name="config_path" value="$(arg config_path)"/>
    </node>

</launch>

